src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -1129,6 +1131,155 @@ TEST_F(HMICapabilitiesTest,
   EXPECT_FALSE(file_system::FileExists(file_cache_name_));
 }
 
+TEST_F(HMICapabilitiesTest,
+       SaveCachedCapabilitiesToFile_CacheFileNameNotSpecified_NoNeedToSave) {
+  MockApplicationManager mock_app_mngr;
+  event_engine_test::MockEventDispatcher mock_dispatcher;
+  MockApplicationManagerSettings mock_application_manager_settings;
+  const smart_objects::CSmartSchema schema;
+  const std::string hmi_capabilities_empty_file_name = "";
+  const std::vector<std::string> sections_to_update{hmi_response::language};
+
+  EXPECT_CALL(mock_app_mngr, event_dispatcher())
+      .WillOnce(ReturnRef(mock_dispatcher));
```

Comment:
This is ON_CALL

Reply:
@SNiukalov change to ON_CALL [here](https://github.com/smartdevicelink/sdl_core/pull/3300/commits/c7a86250ddd5c74c2ba0c2df4d3370977d44e4b5)